test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Audit/AuditTests.cs

```diff
@@ -21,8 +21,11 @@
 
 namespace Microsoft.Health.Fhir.Tests.E2E.Rest.Audit
 {
+    /// <summary>
+    /// Provides Audit specific tests.
+    /// </summary
     [HttpIntegrationFixtureArgumentSets(DataStore.CosmosDb, Format.Json)]
-    public class AuditTests : IClassFixture<AuditTestFixture>
+    public partial class AuditTests : IClassFixture<AuditTestFixture>
```

Comment:
Why is this class now partial?

Reply:
Because of data issue in stu3, the returned status codes for few entries in the bundle are different from R4. So leveraging partial class functionality,  AuditTest for the Batch is written specificly for R4.

Reply:
Does that mean we are not supporting Batch for Stu3? If we do support for Stu3, can we have specific tests written for Stu3 as well?